tracker/tracker_debugger_api.yaml

```diff
@@ -0,0 +1,876 @@
+openapi: 3.0.2
+info:
+  title: Processor Storage API
```

Comment:
Please change to Tracker API

Reply:
Using `Tracker API` would introduce another term after `Processor Storage API` and `Tracker debugger API`. It is better to user `Tracker debugger API` everywhere in this doc.